chart/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
```

Comment:
is this helm api version or charts ?

Reply:
This is auto-generated by `helm create`, I don't really know TBH.

Reply:
BTW is this tested on helm v2 or v3 ?

Reply:
Helm v3 only. Helm v2 is EOL as far as I understand.